core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/BoundedRangeFileInputStream.java

```diff
@@ -93,15 +89,18 @@ public int read(final byte[] b, final int off, int len) throws IOException {
     if (n == 0)
       return -1;
     Integer ret = 0;
-    final FSDataInputStream inLocal = in;
```

Comment:
Is there a reason that the class member `in` can't be marked as `final`? If we're synchronizing on it, that's a good compile-time check to verify that we don't accidentally start synchronizing on different objects.